etl/RetrosheetParser.py

```diff
@@ -60,25 +56,16 @@ def parse_id(self, retro_list):
                 }
         return id_instance
         
-#         # Why not this? Is it more an aesthetic than a practical thing?
```

Comment:
This is a good example of something to discuss with inline comments, as above. As for why not what you wrote in comments - it won't work if all that's passed into the parser is a simple list. The x[1] element of the retro_list is a basic data type like an int or a string, not a list.